ovirt_imageio/_internal/backends/http.py

```diff
@@ -54,8 +54,9 @@ def __init__(self, url, cafile=None, secure=True, connect_timeout=10,
         log.debug("Open netloc=%r path=%r cafile=%r secure=%r",
                   url.netloc, url.path, cafile, secure)
         self.url = url
+        self._is_ssl = url.scheme == "https"
```

Comment:
I had it like that and then added the member variable in a second iteration, to avoid code repetition and the hardocded string in multiple places.

But is true that it is only used twice, so I will change it.